airbyte-cdk/python/pyproject.toml

```diff
@@ -93,13 +92,13 @@ type-check = {cmd = "bin/run-mypy-on-modified-files.sh", help = "Type check modi
 unit-test-with-cov = {cmd = "pytest -s unit_tests -c pytest.ini --cov=airbyte_cdk --cov-report=term --cov-config ../../pyproject.toml", help = "Run unit tests and create a coverage report."}
 # TODO: find a version of the modified mypy check that works both locally and in CI.
 check-local = {sequence = ["lint", "type-check", "unit-test-with-cov"], help = "Lint all code, type-check modified files, and run unit tests."}
-check-ci = {sequence = ["lint", "unit-test-with-cov"], help = "Lint and run unit tests. Does not include type-checking."}
+check-ci = {sequence = ["build", "lint", "unit-test-with-cov"], help = "Build the package, lint and run unit tests. Does not include type-checking."}
 
 # Build and check
 pre-push = {sequence = ["build", "check-local"], help = "Run all build and check tasks."}
 
 [tool.airbyte_ci]
 optional_poetry_groups = ["dev"]
 poetry_extras = ["file-based", "sphinx-docs", "vector-db-based"]
-poe_tasks = ["build", "check-ci"]
+poe_tasks = ["check-ci"]
```

Comment:
Move `build` into `check-ci` because they should happen sequentially (`poe_tasks` happen concurrently). This might not be an issue even, but the generation takes longer than before, so good to be sure. We always want to make sure we run `generate` before we run tests to accurately represent what will be published (this is something we could probably handle better another time by e.g. not checking in the file at all)